### ISSUE TITLE
block import: allow filtering by metro['extent'],  not just city name

### ISSUE THREAD

**Issue by slinkp** [openplans/openblock#154]
The --city argument to import_blocks.py doesn't work well for people who are working in something other than a single urban city, eg. trying to load blocks for multiple towns in a county from a county shapefile.
Proposal from https://groups.google.com/group/ebcode/msg/fb49929f0c41951a

```
Luke just suggested to me that we could add an option to
filter based on a bounding box rather than a city name

Maybe something like:

  import_blocks.py --city-extent=boston

This would work like: find a metro in settings.METRO_LIST whose
short_name is 'boston'; get its extent (minimum & maximum
latitude/longitude); and then filter out blocks that aren't within
that latitude/longitude range.  If you have multiple metros, this
option could be given more than once, eg.

 import_blocks.py --city-extent=boston --city-extent=cambridge

It would also be pretty easy, I think, to allow multiple --city
arguments, so blocks that match any of the named cities would be
allowed in. 
```

**Comment by slinkp** [comment]
we already do something similar in ebdata/ebdata/scrapers/general/open311/georeportv2.py

**Comment by slinkp** [comment]
this is done on my olwidget branch but i don't want to untangle it from that pile of changes yet. it can wait.
... well, mostly done; it only uses the default metro.  But I don't think METRO_LIST is actually useful for anybody other than everyblock.com anyway.

**Comment by slinkp** [comment]
Milestone Beta Release 1 deleted

**Comment by slinkp** [comment]
We've now implemented both --filter-bounds (uses default metro) and --filter-location typeslug:locslug
which allows filtering out stuff outside of one or more Locations.

The proposal as written was predicated on a faulty understanding of multiple-city setups: we don't actually put multiple cities in METRO_LIST.

**Comment by slinkp** [comment]
(In [e667d341925c4058c5667b1485ed9b36e4c5ae5e]) Add a filter_location option to block import. Closes #160

**Comment by slinkp** [comment]
Ticket imported from Trac:
 http://developer.openblockproject.org/ticket/160
Reported by: slinkp
